profile.php

```diff
@@ -201,21 +199,88 @@ function userProfile() {
 			"-> <span style='color:#00C000;'>" . ($player->regen_rate + $player->regen_boost - $regen_cut) . "</span>";
 		}
 		else if(isset($regen_cut)) {
-		
+
 		}
 		echo "<br />";
+    // First attempt:
+    // echo "<label style='width:9.2em;'>Regen Timer:</label>" . (time() - $player->last_update - 60) * -1;
+    $time_since_last_regen = time() - $player->last_update;
 
-        // First attempt:
-        // echo "<label style='width:9.2em;'>Regen Timer:</label>" . (time() - $player->last_update - 60) * -1;
+		//regen timer script - can be moved to its own script.js file
+		echo "
+		<script>
+		var remainingtime = ". 59 - $time_since_last_regen .";
+
+		var health = {$player->health};
+		var max_health = {$player->max_health};
+
+		var chakra = {$player->chakra};
+		var max_chakra = {$player->max_chakra};
+
+		var stamina = {$player->stamina};
+		var max_stamina = {$player->max_stamina};
+
+		var regen = {$player->regen_rate} + {$player->regen_boost}; //no regen cut
+
+		setInterval(() => {
+
+			document.getElementById('regentimer').innerHTML = remainingtime; //minus 1 to compensate for lag
+
+
+			if(remainingtime <= 0){
+				remainingtime = 60;
+
+				if((health + regen) >= max_health){
+					health = max_health;
+				} else {
+					health += regen; //health ignores regen boost
+				}
+
+				if((chakra + regen) >= max_chakra){
+					chakra = max_chakra;
+				} else {
+					chakra += regen;
+				}
+
+				if((stamina + regen) >= max_stamina){
+					stamina = max_stamina;
+				} else {
+					stamina += regen;
+				}
+
+			//update health amounts / bars
+			document.getElementById('health').innerHTML = health.toFixed(2) + '/' + max_health.toFixed(2);
+			document.getElementById('healthbar').style.width = ( health / max_health )*100 + '%';
+
+			document.getElementById('chakra').innerHTML = chakra.toFixed(2) + '/' + max_chakra.toFixed(2);
+			document.getElementById('chakrabar').style.width = ( chakra / max_chakra )*100 + '%';
+
+			document.getElementById('stamina').innerHTML = stamina.toFixed(2) + '/' + max_stamina.toFixed(2);
+			document.getElementById('staminabar').style.width = ( stamina / max_stamina )*100 + '%';
+			}
+
+			remainingtime--;
+
+		}, 1000);
+
+		//for some reason every other tick the javascript regen is ahead of the actual regen?
+		//can't figure out why? its like the Regen changes every other minute in intervals or it doubles
+		//can't find the error with my script
+		//can't seem to find out where the error is, need help.
+
+		</script>
+		";
+
+    echo "<label style='width:9.2em;'>Regen Timer:</label>
+		<p id='regentimer' style='display:inline'>" . (60 - $time_since_last_regen) . "</p>

```

Comment:
```suggestion
		<span id='regentimer'>" . (60 - $time_since_last_regen) . "</span>

```
If we're going to use `display:inline`, may as well use a span here